pypesto/engine/base.py

```diff
@@ -13,15 +13,15 @@ def __init__(self):
 
     @abc.abstractmethod
     def execute(
-        self, tasks: List[Task], progress_bar: bool = True
-    ) -> List[Any]:
+        self, tasks: list[Task], progress_bar: bool = True
+    ) -> list[Any]:
```

Comment:
Is there any difference in using `list` or `List`?

Reply:
Same effect, but `List` and the like are deprecated as of python3.9 (https://docs.python.org/3/library/typing.html#deprecated-aliases)